test/test_helper.ts

```diff
@@ -279,15 +320,20 @@ class TestHelper {
         return {
             name,
             display_name: `Unit Test ${name}`,
-            type: 'O',
+            type: 'O' as TeamType,
```

Comment:
These can be solved with `as const`.

Reply:
true, but we have the `TeamType` definition anyway

Reply:
The difference is that `const foo = 'whatever' as TeamType` does not throw any type warning, while `const foo: TeamType = 'whatever' as const` will.